src/scancode/cli.py

```diff
@@ -131,6 +131,20 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
+
+    scancode --ignore README samples/
+
+Scan a directory while ignoring all files with txt extension. Print scan results to
+stdout as JSON:
+
+    scancode --ignore *.txt samples/
```

Comment:
Thanks for pointing that out, I was doing most of the test on BASH which by default did not expand globs, but it failed in a ZSH. I'll change the examples to use quoted patterns.
Perhaps adding a note about the quotes would help in the `examples_text` doc?